packages/store/src/internal/state-operations.ts

```diff
@@ -34,6 +35,8 @@ export class InternalStateOperations {
   }
 
   private verifyDevMode() {
+    if (isAngularInTestMode()) return;
```

Comment:
@markwhitfeld @eranshmil 

I think this is a difficult operation if the number of iterations in this method is more than it seemed. I think this is a difficult operation if the number of iterations in this method is more than it seemed.

{
 developmentMode: false,
 silentWarning: true // disabled recommend
}

Reply:
I do not want to require developers to add options to all their test setups. This would not be a good experience at all. My aim is to reduce the boilerplate required in the tests.